install-from-mirror.sh

```diff
@@ -35,5 +39,6 @@ pip -vvv install \
     --index-url http://localhost:9090/simple \
     --upgrade \
     "${toplevel}"
+pip freeze
```

Comment:
Might be nice to do include [deptree](https://pypi.org/project/deptree/) output here